tests/test-uptime-check.yml

```diff
@@ -1,45 +1,42 @@
 ---
+
 - name: Test uptime check functionality
   hosts: localhost
   connection: local
   gather_facts: No
   vars:
     apikey: "{{ vault_apikey }}"
     check_name: "OpenshiftAnsibleCheckTest created from {{ inventory_hostname }}"
-    check_protocol: http
-    check_tags: ocp-test,pypingdom-test
+    check_protocol: "http"
+    check_tags: "ocp-test,pypingdom-test"
     check_timing: "60"
+    check_port: "8080"
+    check_encryption: "true"
+    check_certificate: "true"
+    check_filters: "region:EU"
+    check_contain: "google"
+    check_ids: "88522"
```

Comment:
Please can you remove this ID, as it is specific to our Pingdom account.

Reply:
Removed the ID